src/screens/ChatInfoScreen/ChatInfoScreen.tsx

```diff
@@ -310,7 +310,8 @@ const ChatInfoScreen = () => {
         <View style={styles.lineTop} />
         <TouchableOpacity
           testID="imageClick"
-          onPress={handleOnImageClicked}
+          activeOpacity={channelInfo?.channelType === CHANNEL_GROUP ? 0.2 : 1}
+          onPress={channelInfo?.channelType === CHANNEL_GROUP ? () => handleOnImageClicked() : null}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are good for user experience. It prevents users from clicking on the image in a group chat, which is a sensible restriction. However, it's better to move the condition check inside the `handleOnImageClicked` function. This way, you can encapsulate the logic within the function itself and keep the JSX cleaner.
```diff
- activeOpacity={channelInfo?.channelType === CHANNEL_GROUP ? 0.2 : 1}
- onPress={channelInfo?.channelType === CHANNEL_GROUP ? () => handleOnImageClicked() : null}
+ activeOpacity={1}
+ onPress={() => handleOnImageClicked()}
```
And then in your `handleOnImageClicked` function:
```diff
function handleOnImageClicked() {
+   if (channelInfo?.channelType !== CHANNEL_GROUP) {
+     return;
+   }
  // existing code...
}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->